CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0-rc.13] - 2023-05-18
+### Fixed
+- `@distributedlab/fetcher` - fixed error building in response
```

Comment:
avoid using "fixed", "added", or "changed" words in the specific section of the change log coz the section is self-descriptive - it's obvious that u fixed something if u add change under `###Fixed` heading